### DISCUSSION TITLE
Facing Error

### DISCUSSION THREAD

**Discussion by rahmananwar253** [AammarTufail/machinelearning_ka_chilla#5] [Q&A]
when I execute a single line in VS Code it shows output correct, but when I run the whole program it generates a syntax error.

.

**Comment by rasheed2063** [comment]
There might be any spelling mistake, comma missing in your program.
 Please keep in mind that Python runs cells in order so you might not have run the previous cells.
You can "Run All" if you want to run the whole program.

**Comment by Faizali100** [comment]
Share Print screen of error.
